### ISSUE TITLE
Hold (rather than disable) notifications

### ISSUE THREAD

**Issue by fwkoch** [aranzgeo/properties#138]
With #123, we have the ability to disable notifications. However, we do not have the ability to hold notifications until a state is set, then fire them after.

**Comment by fwkoch** [comment]
I think this issue came about from looking at `traitlets` - their widget framework can lead to many traits being modified rapidly at once. In that case, holding notifications is useful. There are no race conditions during interaction, just the final notifications when all the widget dust settles. However, `properties` is designed with more of a focus on direct user interactions, with single, slow, non-conflicting updates.

There is still a possibility multiple properties are set at once through kwargs - this creates ambiguity for the order of notifications (if you set `({'a': ..., 'b': ...})` does 'a' notification or 'b' notification fire first?). But holding notifications does not provide a solution to that, it only postpones the problem.

Bottom line is I do not think we need to hold notifications.
